### PR TITLE
Allow TLS 1.3 CertificateVerify to handle 8192-bit RSA

### PR DESCRIPTION
**Pull request by SparkiDev**

### CODE REVIEW

File: wolfssl/internal.h [hunk 1]
```diff
@@ -1176,6 +1176,14 @@ enum {
     #define MAX_EARLY_DATA_SZ  4096
 #endif
 
+#ifndef WOLFSSL_MAX_RSA_BITS
+    #if (defined(USE_FAST_MATH) && defined(FP_MAX_BITS) && FP_MAX_BITS >= 16384)
+        #define WOLFSSL_MAX_RSA_BITS 8192
```
Comment (grooverdan) [inline wolfssl/internal.h:1181]: Although 8k is a bit crazy already - shouldn't this be:
<pre>
#define WOLFSSL_MAX_RSA_BITS  (FP_MAX_BITS / 2 )
</pre>
For the even crazier people that want even bigger.

Comment (SparkiDev) [reply wolfssl/internal.h]: Thanks @grooverdan!
Fixed.

**Comment by grooverdan** [comment]
Great work, works in the mariadb codebase too:

main.ssl_8k_key                          [ pass ]    227

However I'd like to see some server protection against a malicious user providing a too large certificate. I can't tell if the point at which the stack/heap corruption occurs before or after the CA validation however I think the server code should be robust enough that even a signed >8k cert is presented, it shouldn't crash the server. I can't see this done for the <tls1.3 case either (in comparisons against MAX_CERT_VERIFY_SZ).

Aside - in SendTls13CertificateVerify, MAX_MSG_EXTRA is added to the sendSz twice. Looks like an error.

Thank you so much for fixing this bug.

**Comment by grooverdan** [comment]
ah, verification was in ENCRYPT_LEN that you just added right?

**Comment by SparkiDev** [comment]
The crash was in the construction of the CertificateVerify message. Not in parsing.
ENCRYPT_LEN was also updated as it was one of the other places I saw FP_MAX_BITS being checked.

**Comment by grooverdan** [comment]
other trivial suggestion is to validate FP_MAX_BITS is a multiple of 8 like WOLFSSL_MAX_DHKEY_BITS.

**Comment by SparkiDev** [comment]
Thanks @grooverdan!
Pull request updated with your suggestion.

**Pull request merged**